### PR TITLE
Solitary-Snipe Fixes For Jetstream

### PR DESCRIPTION
**Pull request by amercer1**
Highlight Reel:

- [x] Update atmosphere-ansible to the latest from [upstream repo](https://github.com/iPlantCollaborativeOpenSource/atmosphere-ansible)
- [x] Mount volumes to use ext4 by default
- [x] DHCP exit hook to *not* use external IP resolution services by default
- [x] Remove ntp configuration as a step 
- [x] Variable-ize fail2ban and adjust `max_retry`, `findtime`, and `bantime` defaults

### CODE REVIEW

File: ansible/roles/atmo-novnc/vars/CentOS-5.yml [hunk 1]
```diff
@@ -51,11 +51,11 @@ REQUIRED:
...
-  - /usr/bin/vnclicense -add {{ VNCLICENSE }}
-  - /bin/su {{ ATMOUSERNAME }} -c /usr/bin/vncserver :1 -vncconfigfile /home/{{ ATMOUSERNAME }}/.vnc/config.custom
   - "for i in $(/usr/bin/pgrep -f '/opt/kanaka-noVNC-8b0a0f6/utils/');do /bin/echo 'killing process:' $i; /bin/kill -HUP $i;done"
-  - "/bin/su {{ ATMOUSERNAME }} -c '/usr/bin/vncserver -localhost :{{ NOVNC.vnc_session }} -vncconfigfile {{ NOVNC.config_file }}'"
+  - /usr/bin/vnclicense -add {{ VNCLICENSE }}
+  - /bin/su {{ ATMOUSERNAME }} -c /usr/bin/vncserver :1 
+  #- "/bin/su {{ ATMOUSERNAME }} -c '/usr/bin/vncserver -localhost :{{ NOVNC.vnc_session }} -vncconfigfile {{ NOVNC.config_file }}'"
   # To disable NoVNC web when entering prod, remove "--web /usr/share/novnc/" from the command below
-  - "/opt/kanaka-noVNC-8b0a0f6/utils/launch.sh --listen {{ NOVNC.socket_port }} --vnc localhost:{{ NOVNC.vnc_port }} &"
+  #- "/opt/kanaka-noVNC-8b0a0f6/utils/launch.sh --listen {{ NOVNC.socket_port }} --vnc localhost:{{ NOVNC.vnc_port }} &"
```
Comment (steve-gregory) [inline ansible/roles/atmo-novnc/vars/CentOS-5.yml:61]: Where are these ports defaults listed?

**Review by steve-gregory — APPROVED** [review]
Looks good, in the future I would suggest bringing in upstream changes separate from making playbook changes.

File: ansible/roles/atmo-novnc/vars/CentOS-5.yml (hunk 1 above)
Comment (amercer1) [reply ansible/roles/atmo-novnc/vars/CentOS-5.yml:61]: In the defaults [file](https://github.com/jetstream-cloud/atmosphere-ansible/blob/integrate-mount-roles/ansible/roles/atmo-novnc/defaults/main.yml).

Comment (steve-gregory) [reply ansible/roles/atmo-novnc/vars/CentOS-5.yml:61]: Perfect!

**Comment by steve-gregory** [comment]
Adjusting title and description to properly reflect the new scope of the PR

**Comment by steve-gregory** [comment]
:shipit: Thanks again @amercer1 !

**Pull request merged**